falcon/app.py

```diff
@@ -766,16 +766,13 @@ def set_error_serializer(self, serializer):
             def my_serializer(req, resp, exception):
                 representation = None
 
-                preferred = req.client_prefers(('application/x-yaml',
-                                                'application/json'))
+                preferred = req.client_prefers((MEDIA_YAML, MEDIAL_JSON))
```

Comment:
It might also make sense to refer to these constants using the `falcon.` namespace, as otherwise we are implying that the reader has to `from falcon import *`.